src/__swaps__/screens/Swap/hooks/useSwapInputsController.ts

```diff
@@ -12,25 +12,92 @@ import {
   countDecimalPlaces,
   extractColorValueForColors,
   findNiceIncrement,
+  isUnwrapEth,
+  isWrapEth,
   niceIncrementFormatter,
+  priceForAsset,
   trimTrailingZeros,
   valueBasedDecimalFormatter,
 } from '@/__swaps__/utils/swaps';
-import { ChainId } from '@/__swaps__/types/chains';
+import { ChainId, ChainName } from '@/__swaps__/types/chains';
 import { ParsedSearchAsset } from '@/__swaps__/types/assets';
 import { useColorMode } from '@/design-system';
 import { isSameAssetWorklet } from '@/__swaps__/utils/assets';
+import {
+  CrosschainQuote,
+  ETH_ADDRESS,
+  Quote,
+  QuoteError,
+  QuoteParams,
+  Source,
+  SwapType,
+  getCrosschainQuote,
+  getQuote,
+} from '@rainbow-me/swaps';
+import { chainNameFromChainIdWorklet } from '@/__swaps__/utils/chains';
+import { RainbowConfig, useRemoteConfig } from '@/model/remoteConfig';
+import { useAccountSettings } from '@/hooks';
+import { convertAmountToRawAmount, convertRawAmountToBalance, convertRawAmountToNativeDisplay } from '@/__swaps__/utils/numbers';
+import ethereumUtils from '@/utils/ethereumUtils';
+import { FormattedExternalAsset, fetchExternalToken } from '@/resources/assets/externalAssetsQuery';
+
+const QUOTE_REFETCH_INTERVAL = 5_000;
+const PRICE_REFETCH_INTERVAL = 10_000;
+
+export const DEFAULT_SLIPPAGE_BIPS = {
+  [ChainId.mainnet]: 100,
+  [ChainId.polygon]: 200,
+  [ChainId.bsc]: 200,
+  [ChainId.optimism]: 200,
+  [ChainId.base]: 200,
+  [ChainId.zora]: 200,
+  [ChainId.arbitrum]: 200,
+  [ChainId.avalanche]: 200,
+  [ChainId.blast]: 200,
+};
+
+export const DEFAULT_SLIPPAGE = {
```

Comment:
having both of these seems redundant

Reply:
agreed. will just store the default and convert to bips when needed.